src/index.css

```diff
@@ -2,16 +2,76 @@
   font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
   line-height: 1.5;
   font-weight: 400;
-
+  
   color-scheme: light dark;
   color: rgba(255, 255, 255, 0.87);
   background-color: #242424;
-
+  
   font-synthesis: none;
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+.dash-logo{
+  width: 180px;
+  height: 140px;
+}
+
+.project-logo{
+  width:200px;
+  height: 100px;
+  margin-left: 10rem;
+}
+
+.avatar-img{
+  width: 50px;
+  height: 50px;
+}
+
+.forget-password{
+  font-size: 18px;
```

Comment:
We shouldn't be using pixel for font size as it has accessibility issues such as not increasing when the user change the browser's default font-size. We should be using rem instead.